src/components/Calculator/Calculator.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Button from '../Button/Button';
+import './calculator.css';
+
+const Calculator = () => {
+  const operatorsBtnColor = '#f4b848';
+  return (
+    <div className="calculator-wrapper">
+      <div className="display">0</div>
```

Comment:
- [ ] `Calculator.js` should also be a class based component, kindly implement this as well